config-files/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### PR Checklist
+
+- [ ] JIRA Ticket(s) included in PR Title and are formatted correctly if applicable (JIRA-123)
```

Comment:
Though `Applicable JIRA Ticket(s) included in the PR Title and formatted correctly: JIRA-123` is short, simple and they can just check the box.